analytics_data_api/v0/views/learners.py

```diff
@@ -242,8 +258,8 @@ def get_queryset(self):
 
 class CourseLearnerMetadata(CourseViewMixin, generics.RetrieveAPIView):
     """
-    Get metadata on learners within a course. Includes data on segments,
-    cohorts, enrollment modes, and an engagement rubric.
+    Get metadata for the learners in a course. Includes data on segments,
```

Comment:
It's data about learners in a course. It's not exactly for the learners themselves to see.